src/table/TableWrapper.jsx

```diff
@@ -51,6 +62,33 @@ export default function TableWrapper(props) {
     return null;
   }
 
+  useEffect(() => {
+    window.addEventListener('keydown', (event) =>
```

Comment:
Not sure we want to put the listener here. I was thinking it could use the one on the cells, but I guess you would still want it to work if you have focus on the rows per page dropdown for example. So i guess then it needs to be on the whole table. that being said, it should be the component, not on the window.

Reply:
This also opens up the discussion if we should have the navigation and selections seperated on different listeners. We will have to have a listener on a higher level for this to work, it might be cleaner to have the navigation there as well. Let's stick to this for now, and maybe consider a refactor if we run in to trouble or something